<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>librarian_window</class>
 <widget class="QMainWindow" name="librarian_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>your library</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>681</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>orders</string>
     </attribute>
     <widget class="QTableWidget" name="pending_orders_table">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>661</width>
        <height>391</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <column>
       <property name="text">
        <string>order id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>books</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>status</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>pending orders:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>penalty is:</string>
      </property>
     </widget>
     <widget class="QLabel" name="penalty_label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>450</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>books</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>171</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>genre</string>
      </property>
      <widget class="QLineEdit" name="genre_name_edit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>name</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add_genre_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>121</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>add new genre</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>author</string>
      </property>
      <widget class="QPushButton" name="add_author_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>141</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>add new author</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="author_name_edit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>161</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>name</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>281</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>books</string>
      </property>
      <widget class="QPushButton" name="add_book_button">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>310</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>add new book</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="authors_table">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>261</width>
         <height>91</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <column>
        <property name="text">
         <string>author</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
      <widget class="QTableWidget" name="genres_table">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>261</width>
         <height>81</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>120</number>
       </attribute>
       <column>
        <property name="text">
         <string>genre</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>250</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>penalty per day:</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="slider_label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>290</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QLabel" name="slider_label_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>290</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>rub</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="book_name_edit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="maxLength">
        <number>50</number>
       </property>
       <property name="placeholderText">
        <string>name</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>150</y>
        <width>301</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>add book instances </string>
      </property>
      <widget class="QPushButton" name="add_book_instance_button">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>170</y>
         <width>191</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>add new book instance</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>book id:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="book_id_combobox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>104</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="condition_combobox">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>70</y>
         <width>141</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>condition:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>book instance id:</string>
       </property>
      </widget>
      <widget class="QLabel" name="bi_id_label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>380</y>
        <width>191</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>remove book instances</string>
      </property>
      <widget class="QComboBox" name="book_to_remove_combobox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>104</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>60</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>id:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="remove_book_button">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>113</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>remove</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>slider_label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>433</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
